### DISCUSSION TITLE
Is the r2modman-3.1.45.exe in releases (not the setup exe) for portable use?

### DISCUSSION THREAD

**Discussion by tylerwight** [ebkr/r2modmanPlus#1157] [Q&A] [answered]
https://github.com/ebkr/r2modmanPlus/releases/download/v3.1.45/r2modman-3.1.45.exe 

I have searched around for confirmation on this and not seeing anything specific. I mean I guess if it works then it works, I'm just curious on the actual intent.

When downloading the above .exe I can download it and run it and it seems to work just fine, no setup/install required. But When you install with the setup there is a lot more files (dll files, etc. ) in the r2modman folder. I am hoping for a portable version cause I don't like to install things.

(as opposed to this exe https://github.com/ebkr/r2modmanPlus/releases/download/v3.1.45/r2modman-Setup-3.1.45.exe)

**Comment by ebkr** [comment, marked as answer]
The non-setup exe is for portable use. It's not really recommended as it doesn't stay updated, but it is viable for when you don't want to / can't install it normally.
